settings.ini

```diff
@@ -49,7 +49,7 @@ doc_host =  https://%(user)s.github.io
 #doc_host = https://pages.github.%(company_name)s.com.  
 
 
-doc_baseurl = /%(lib_name)s/
```

Comment:
I think this might cause issues.  I added it because without it, the jekyll serving showed `Server address: http://0.0.0.0:4000/nbverbose//` so I thought it was redundant, but without it, I'm seeing issues with the doc links
(i.e `<a href="/nbverboseshowdoc.html#show_doc">`) Missing the /

Reply:
(fixed this I believe)